blaze/compute/core.py

```diff
@@ -367,20 +367,20 @@ def compute(expr, d, return_type=no_default, **kwargs):
         The blaze expression to compute.
     d : any
         The data source to compute expression on.
-    return_type : {'native', 'core', type}, optional
-        Type to return data as. Defaults to 'native' but will be changed
-        to 'core' in version 0.11.  'core' forces the computation into a core
-        type. 'native' returns the result as is from the respective backend's
-        ``post_compute``. If a type is passed, it will odo the result into the
-        type before returning.
+    return_type : {'core', 'native', type}, optional
+        Type to return data as.  Defaults to 'core' since version 0.11.1.
+        'core' forces the computation into a core type. 'native' returns
+        the result as is from the respective backend's ``post_compute``.
+        If a type is passed, it will odo the result into the type before
+        returning.
 
     Examples
     --------
     >>> t = symbol('t', 'var * {name: string, balance: int}')
     >>> deadbeats = t[t['balance'] < 0]['name']
 
     >>> data = [['Alice', 100], ['Bob', -50], ['Charlie', -20]]
-    >>> list(compute(deadbeats, {t: data}))
+    >>> list(compute(deadbeats, {t: data}, return_type='native'))
```

Comment:
is `return_type = 'native'` necessary here?  I would have expected `return_type = 'core'` to return a `list`.